packages/@sanity/desk-tool/src/utils/resolvePanes.js

```diff
@@ -297,3 +302,28 @@ function isStructure(structure) {
       typeof structure.type !== 'string')
   )
 }
+
+function warnOnUnknownExports(mod) {
+  if (!mod) {
+    return
+  }
+
+  const known = KNOWN_STRUCTURE_EXPORTS.concat('default')
+  const keys = Object.keys(mod)
+  keys
+    .filter(key => !known.includes(key))
+    .forEach(key => {
+      const {closest} = known.reduce(
+        (acc, current) => {
+          const distance = leven(current, key)
+          return distance < 3 && distance < acc.distance ? {closest: current, distance} : acc
+        },
+        {closest: null, distance: +Infinity}
+      )
+
+      const hint = closest ? ` - did you mean "${closest}"` : ''
```

Comment:
Sometimes, when I read nice code, it makes me happy. This code makes me happy :)